src/backend/alacritty.rs

```diff
@@ -114,15 +112,19 @@ impl Alacritty {
 
 impl Functions for Alacritty {
     fn create_command(&mut self, config: &Config) -> std::process::Command {
-        let mut base_conf = match config.load_term_conf {
-            true => Alacritty::load_alacritty_conf(),
-            _ => serde_yaml::Mapping::new(),
-        };
-
-        self.create_conf_file(&mut base_conf, config);
-        let mut command = std::process::Command::new(self.exe_path.to_owned());
-        command.arg("--config-file");
-        command.arg(self.cfg_file.as_ref().unwrap().path());
+        let mut command = std::process::Command::new(&self.exe_path);
+        if let Some(config_path) = config.term_config_path.as_ref() {
+            command.arg("--config-file");
+            command.arg(config_path);
+        } else {
+            let mut base_conf = match config.load_term_conf {
+                true => Alacritty::load_alacritty_conf(),
```

Comment:
Same reason as the comment for kitty, instead of loading the `term_config_path` here, can we pass it as a param for `load_alacritty_conf()` so it can be loaded as the base config and `glrnvim` can overwrite other settings in the `glrnvim.yml`?